python/az/aro/azext_aro/custom.py

```diff
@@ -133,15 +133,14 @@ def aro_list_credentials(client, resource_group_name, resource_name):
     return client.list_credentials(resource_group_name, resource_name)
 
 
-def aro_update(client, resource_group_name, resource_name, worker_count=None,
-               no_wait=False):
+# Currently this function will not trigger any changes
+def aro_update(client, resource_group_name, resource_name, no_wait=False):
```

Comment:
I somehow think we should just remove dead code :man_shrugging:

Reply:
I am in split mind regarding this too. I followed instructions in the ticket which specifically says to leave the function for now and let it do nothing. 
I think this is early stages and we might need update functionality anyways. Plus on top of that removing "update" code from backend involves many more changes then removing one file. (so restoring it would be also a lot of work)

Reply:
trim this down to the following for now?
```
def aro_update(client, resource_group_name, resource_name, no_wait=False):
    oc = v2019_12_31_preview.OpenShiftCluster()

    return sdk_no_wait(no_wait, client.update,
                       resource_group_name=resource_group_name,
                       resource_name=resource_name,
                       parameters=oc)
```

Reply:
bump